pom.xml

```diff
@@ -240,7 +252,7 @@
                         </goals>
                         <configuration>
                             <compilerArgs>
-                                <arg>-Xlint:unchecked,deprecation</arg>
+                                <arg>-Xlint:unchecked</arg>
```

Comment:
Why remove deprecation?

Reply:
Huh, thanks for pointing it out. This is another problem we have to solve. Test container from `1.5.x` version deprecated standard `GenericContainer` constructor. In our case, it's `StrimziKafkaContainer` and `StrimziZookeeperContainer` and the possible re-placement is to use a constructor where the user has to specify `DockerImageName`, which is what we want to avoid... ))

Reply:
Well, the question is whether disabling it here is the right approach. I think you should keep it here enabled and instead use annotation to suppress the deprecation warning where needed. That is what we follow in operators etc.

Reply:
Make sense :) Will do